### PR TITLE
Improve test_KEGG.py coverage

### PR DESCRIPTION
**Pull request by Gasta88**
This pull request addresses issue #1288 

I hereby agree to dual licence this and any previous contributions under both
the _Biopython License Agreement_ AND the _BSD 3-Clause License_.

I am happy to be thanked by name in the ``NEWS.rst`` and
``CONTRIB.rst`` files.

I have read the ``CONTRIBUTING.rst`` file and understand that AppVeyor and
TravisCI will be used to confirm the Biopython unit tests and ``flake8`` style
checks pass with these changes.

### CODE REVIEW

File: Tests/test_KEGG.py [hunk 1]
```diff
@@ -58,6 +58,11 @@ def test_sample(self):
                           ('BRENDA, the Enzyme Database', ['1.1.1.1']),
                           ('CAS', ['9031-72-5'])])
         self.assertEqual(records[-1].entry, "2.7.2.1")
+        self.assertEqual(records[-1].__str__().replace(" ", "").split("\n")[:10],
```
Comment (peterjc) [inline Tests/test_KEGG.py]: Rather than calling the magic method explicitly,  I would just use ``str(records[-1])`` here.

**Comment by peterjc** [comment]
It looks like the TravisCI tests are failing on an external dependency...

**Comment by Gasta88** [comment]
Is there any way that I can resolve the `PhyML` issue? Or is it concerning TravisCI only?

**Comment by peterjc** [comment]
By PhyML issue I assume you mean the DSSP problem? I've committed fda9116365bd94c5dd8e46d3d20fc78df8cefa43 for that.

If you were to refresh this branch TravisCI should run normally now. e.g. Commit a tiny change like the ``__str__`` edit I suggested.

**Comment by peterjc** [comment]
Thanks!

**Pull request merged**